sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -4,35 +4,36 @@
 
 ```ts
 
-import * as coreHttp from '@azure/core-http';
+import { CommonClientOptions } from '@azure/core-client';
+import { OperationOptions } from '@azure/core-client';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { PollerLike } from '@azure/core-lro';
 import { PollOperationState } from '@azure/core-lro';
-import { TokenCredential } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-auth';
 
 // @public
-export interface AccessControlClientOptions extends coreHttp.PipelineOptions {
+export interface AccessControlClientOptions extends CommonClientOptions {
```

Comment:
Maor and I have talked about whether this is a breaking change. I don’t think so, but Let’s get @xirzec ‘s thoughts

Reply:
There are 3 relatively minor breaking changes:

- Optional property `mode` was removed from `retryOptions` since it only ever had a single enum value and was never used.
- `handleRedirects` was removed from `redirectOptions`, as the policy can now be controlled by removing it from the pipeline or setting `maxRedirects` to 0.
- `keepAliveOptions` were removed. Keep alive can be disabled on a per-request basis with `disableKeepAlive`.

These were all fairly niche options that tended to be used internally by our clients rather than set by consumers, but since we did expose them, we should consider what versioning implication this poses. 

Alongside the change to remove `_response`, a strict semver interpretation would be to major version the package, but debatably a minor bump could be sufficient. /cc @chradek @ramya-rao-a @jeremymeng - I think we should have broad agreement on what our policy is here.

Reply:
I do plan to merge this PR, but leaving this conversation open if folks want to chime in. I'll also add it to the list of team meeting topics I have here 👍

Reply:
Since this package is still a beta I am less worried about it and don't think the discussion is blocking this PR

Reply:
Can we log an issue to ensure the discussion is continued and we come to a conclusion before this package goes GA?

Reply:
Sure, opened #15916 15916